.golangci.yml

```diff
@@ -207,3 +207,8 @@ output:
   formats:
     - format: colored-line-number
       path: stderr
+
+exclude-rules:
+  - path: _test\.go$
+    linters:
+      - forbidigo
```

Comment:
This is in the wrong location. Add forbidigo where we have the other exclusions for test files.